### PR TITLE
z-index support added

### PR DESCRIPTION
**Pull request by enchev**
Implemented #515

### CODE REVIEW

**Comment by enchev** [comment]
run ci

**Comment by vakrilov** [comment]
👍 
However we should state explicitly that this will not work on Android API < 21

File: ui/core/view.android.ts [hunk 1]
```diff
@@ -700,6 +700,23 @@ export class ViewStyler implements style.Styler {
...
     }
+    
+    // z-index
+    private static getZIndexProperty(view: View): any {
+        return view.android.getZ ? view.android.getZ() : 0;
+    }
+
+    private static setZIndexProperty(view: View, newValue: any) {
+        if (view.android.setZ) {
+            view.android.setZ(utils.layout.toDeviceIndependentPixels(newValue));
```
Comment (hshristov) [inline ui/core/view.android.ts]: Converting toDIP - will probably result to non Int value. We need to remove it.

**Comment by vchimev** [comment]
run ci

**Pull request merged**